.travis.yml

```diff
@@ -33,19 +26,28 @@ notifications:
   email: false
 
 before_install:
+- pip3 install --upgrade pip
 - |
   if [ "$TRAVIS_OS_NAME" = "linux" ]; then
     OS=Linux-x86_64
-    sudo apt-get install -q -y libmecab-dev swig mecab unidic-mecab
+    sudo apt-get install -q -y libmecab-dev swig mecab
+    # Binary dictionary not included in 2.2.0-1 (https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=788822)
+    wget http://launchpadlibrarian.net/326401889/unidic-mecab_2.1.2~dfsg-7_all.deb
+    sudo dpkg -i unidic-mecab_2.1.2~dfsg-7_all.deb
     # Set the max number of connections 100
     # No need to adjust it on Mac as it's 100 by default
     sudo sed -i 's/max_connections = 255/max_connections = 100/g' /etc/postgresql/10/main/postgresql.conf
     sudo service postgresql restart
     # Install ImageMagick library for Wand
     sudo apt-get install libmagickwand-dev ghostscript
     sudo rm -rf /etc/ImageMagick-6/policy.xml
+    # Install poppler-utils
+    sudo apt install -q -y poppler-utils
   else
     OS=MacOSX-x86_64
+    pip install virtualenv
```

Comment:
pip3?

Reply:
Thanks. pip actually works fine on OSX.
For better consistency, pip3 is better but Makefile also needs fixing.
Do you mind if I fix Makefile too?

Reply:
How about we use alias to use `pip` for all the time?

Reply:
Okay. Will do.